cf_units/_udunits2_parser/parser/udunits2Lexer.py

```diff
@@ -1,6 +1,6 @@
-# Generated from /home/ruth/git_stuff/cf-units/cf_units/_udunits2_parser/parser/udunits2Lexer.g4 by ANTLR 4.11.1
+# Generated from cf_units/_udunits2_parser/parser/udunits2Lexer.g4 by
```

Comment:
Still changes to auto-generated files - would suggest not making any changes to auto-regeneratable files. (everything in ``_udunits2_parser.parswr``)

Reply:
https://github.com/SciTools/cf-units/pull/364/commits/e0b930c34acdece06f3be4a772ab4bd3f5562391